modules/post/windows/gather/credentials/winbox_settings.rb

```diff
@@ -0,0 +1,83 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::Common
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Gather Mikrotik Winbox "Keep Password" Credentials Extractor',
+        'Description' => %q{
+          This module extracts Mikrotik Winbox credentials saved in the
+          "settings.cfg.viw" file when the "Keep Password" option is
+          selected in Winbox.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => ['Pasquale \'sid\' Fiorillo'], # www.pasqualefiorillo.it - Thanks to: www.isgroup.biz
+        'Platform' => ['win'],
+        'SessionTypes' => ['meterpreter', 'shell'],
```

Comment:
Sorry- verified this works great, so last quick change.
```suggestion
        'SessionTypes' => ['meterpreter', 'shell', 'powershell'],
```